deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -210,6 +210,7 @@ export class StackComposer {
         const otelCollectorEnabled = this.getContextForType('otelCollectorEnabled', 'boolean', defaultValues, contextJSON)
         const reindexFromSnapshotServiceEnabled = this.getContextForType('reindexFromSnapshotServiceEnabled', 'boolean', defaultValues, contextJSON)
         const reindexFromSnapshotExtraArgs = this.getContextForType('reindexFromSnapshotExtraArgs', 'string', defaultValues, contextJSON)
+        const reindexFromSnapshotMaxShardSizeGiB = this.getContextForType('reindexFromSnapshotMaxShardSizeGiB', 'number', defaultValues, contextJSON)
```

Comment:
What are the limits around this? How big can (should) we get? Can we check the bounds and throw an error earlier instead of later?

Reply:
I think the stack will fail to synthesize if you go over the maximum EBS size, but I can add an explicit exception.  Good call.